concepts/src/concept_csv_export.py

```diff
@@ -45,16 +45,19 @@ def set_globals(
     user: Optional[str] = None,
     password: Optional[str] = None,
     version: float = VERSION,
+    locales: list[str] = LOCALES,
```

Comment:
@brandones I get this error when I run the script which works when I remove the datatype definition thus have line 48 as `locales: list = LOCALES,`
`Traceback (most recent call last):
  File "concepts/src/concept_csv_export.py", line 48, in <module>
    locales: list[str] = LOCALES,
TypeError: 'type' object is not subscriptable`
Do you know what might be causing this?

Reply:
Are you running with Python 3? This only works with Python 3.8+ (see the README).